contract/package.json

```diff
@@ -47,7 +47,9 @@
   },
   "dependencies": {
     "@agoric/ertp": "^0.16.3-u12.0",
+    "@agoric/vat-data": "0.5.3-u16.1",
     "@agoric/zoe": "^0.26.3-u12.0",
+    "@agoric/zone": "^0.3.0-u16.1",
```

Comment:
What impact does this have on the contract bundle size? Does it introduce duplicate dependencies?

Other dependencies in this dapp (zoe, ertp) look like they're from u12. IME, mixing u12 and u16 is asking for trouble.

I know dapp-agoric-basics uses u14: https://github.com/Agoric/dapp-agoric-basics/commit/817bc8a2ad118155e4c151dfd59384fecccbebe8

Are you familiar with the fragility of dependency versions?
 - https://github.com/Agoric/agoric-sdk/issues/8621
 - #34
 - #36
 - https://github.com/Agoric/agoric-sdk/issues/7501

It looks like @Jovonni managed to use u16 in dapp-orchestration-basics, though it took some [troubleshooting](https://github.com/Agoric/dapp-orchestration-basics/blob/dev/orca-v1/TROUBLESHOOTING.md). See item 9 in particular.